org.lemonade_emu.lemonade.json

```diff
@@ -0,0 +1,172 @@
+                        "url-template": "https://github.com/KhronosGroup/SPIRV-Headers/archive/refs/tags/sdk-$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "rapidjson",
+            "buildsystem": "cmake-ninja",
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DRAPIDJSON_ENABLE_INSTRUMENTATION_OPT=OFF",
+                "-DRAPIDJSON_BUILD_DOC=OFF",
+                "-DRAPIDJSON_BUILD_EXAMPLES=OFF",
+                "-DRAPIDJSON_BUILD_TESTS=OFF",
+                "-DCMAKE_INTERPROCEDURAL_OPTIMIZATION=ON"
+            ],
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://github.com/Tencent/rapidjson/archive/v1.1.0.tar.gz",
+                    "sha256": "bf7ced29704a1e696fbccf2a2b4ea068e7774fa37f6d7dd4039d0787f8bed98e",
+                    "x-checker-data": {
+                        "type": "anitya",
+                        "stable-only": true,
+                        "project-id": 7422,
+                        "url-template": "https://github.com/Tencent/rapidjson/archive/v$version.tar.gz"
+                    }
+                }
+            ]
+        },
+        {
+            "name": "lemonade",
+            "buildsystem": "cmake-ninja",
+            "builddir": true,
+            "config-opts": [
+                "-DCMAKE_BUILD_TYPE=Release",
+                "-DCMAKE_C_COMPILER=clang",
+                "-DCMAKE_CXX_COMPILER=clang++",
+                "-DCMAKE_LINKER=lld",
+                "-DENABLE_QT_TRANSLATION=ON",
+                "-DLEMONADE_ENABLE_COMPATIBILITY_REPORTING=OFF",
+                "-DUSE_DISCORD_PRESENCE=OFF",
+                "-DENABLE_COMPATIBILITY_LIST_DOWNLOAD=OFF",
+                "-DUSE_SYSTEM_SDL2=ON"
+            ],
+            "cleanup": [
+                "/share/man",
+                "/share/pixmaps"
+            ],
+            "post-install": [
+                "install -Dm755 ../lemonade-launcher.sh /app/bin/lemonade-launcher",
+                "install -Dm644 ../org.lemonade_emu.lemonade.metainfo.xml /app/share/appdata/org.lemonade_emu.lemonade.metainfo.xml",
+                "desktop-file-install --dir=/app/share/applications ../dist/lemonade-qt.desktop",
+                "desktop-file-edit --set-key StartupWMClass --set-value lemonade-qt --set-key Exec --set-value lemonade-launcher /app/share/applications/lemonade-qt.desktop",
+                "install -Dm644 ../dist/lemonade.svg /app/share/icons/hicolor/scalable/apps/lemonade.svg",
+                "install -Dm644 ../dist/icon.png /app/share/icons/hicolor/512x512/apps/lemonade.png",
+                "mv /app/share/mime/packages/lemonade.xml /app/share/mime/packages/org.lemonade_emu.lemonade.xml",
+                "sed 's/lemonade/org.lemonade_emu.lemonade/g' -i /app/share/mime/packages/org.lemonade_emu.lemonade.xml"
```

Comment:
Please see the thread https://github.com/flathub/flathub/pull/5133#discussion_r1552199625

The is of sed is always strongly discourage.